show/plugins/auto_techsupport.py

```diff
@@ -0,0 +1,159 @@
+"""
+Auto-generated show CLI plugin.
+Manually Edited to add show cli for "show auto_techsupport history"
+"""
+
+import click
+import tabulate
+import natsort
+import utilities_common.cli as clicommon
+
+
+def format_attr_value(entry, attr):
+    """ Helper that formats attribute to be presented in the table output.
+
+    Args:
+        entry (Dict[str, str]): CONFIG DB entry configuration.
+        attr (Dict): Attribute metadata.
+
+    Returns:
+        str: fomatted attribute value.
+    """
+
+    if attr["is-leaf-list"]:
+        return "\n".join(entry.get(attr["name"], []))
+    return entry.get(attr["name"], "N/A")
+
+
+def format_group_value(entry, attrs):
+    """ Helper that formats grouped attribute to be presented in the table output.
+
+    Args:
+        entry (Dict[str, str]): CONFIG DB entry configuration.
+        attrs (List[Dict]): Attributes metadata that belongs to the same group.
+
+    Returns:
+        str: fomatted group attributes.
+    """
+
+    data = []
+    for attr in attrs:
+        if entry.get(attr["name"]):
+            data.append((attr["name"] + ":", format_attr_value(entry, attr)))
+    return tabulate.tabulate(data, tablefmt="plain")
+
+
+@click.group(name="auto-techsupport",
+             cls=clicommon.AliasedGroup)
+def AUTO_TECHSUPPORT():
+    """ AUTO_TECHSUPPORT part of config_db.json """
+
+    pass
+
+
+@AUTO_TECHSUPPORT.command(name="global")
+@clicommon.pass_db
+def AUTO_TECHSUPPORT_GLOBAL(db):
```

Comment:
>AUTO_TECHSUPPORT_GLOBAL [](http://example.com/codeflow?start=4&length=23)

Do you have a strong reason to use UPPER_CASE function name? We prefer lower_case.

Reply:
That's auto_generated by SONiC Auto CLI gen. https://github.com/Azure/sonic-utilities/pull/1644